packages/flutter/lib/src/widgets/binding.dart

```diff
@@ -618,27 +610,30 @@ mixin WidgetsBinding on BindingBase, ServicesBinding, SchedulerBinding, GestureB
   /// Tell the framework not to report the frame it is building as a "useful"
   /// first frame until there is a corresponding call to [allowFirstFrameReport].
   ///
-  /// This is used by [WidgetsApp] to avoid reporting frames that aren't useful
-  /// during startup as the "first frame".
+  /// Deprecated. Use [deferFirstFrame]/[allowFirstFrame] to delay rendering the
+  /// first frame.
+  @Deprecated(
+    'Use deferFirstFrame/allowFirstFrame to delay rendering the first frame. '
```

Comment:
```suggestion
    'Use deferFirstFrame/allowFirstFrame to delay rendering the first frame.'
```

Reply:
This space is required because the String continuous on the next line and there should really be a space between the "." of this line and the "This" of the next line. :)